map/search_mark.cpp

```diff
@@ -238,7 +239,9 @@ float SearchMarkPoint::GetSymbolOpacity() const
 
 df::ColorConstant SearchMarkPoint::GetColorConstant() const
 {
-  return kColorConstant;
+  if (IsHotel() && m_stars >= 4)
+    return kGoodHotelRatingColor;
```

Comment:
As I mentioned, it's an unfair advantage for richer hotels. Let's remove it.

Reply:
I don't insist on this change, but we also show stars in search view list and all this PR is about stars.